### DISCUSSION TITLE
Inav 6 Configurator and possibility of Win 32 bit version?

### DISCUSSION THREAD

**Discussion by corycleveland** [iNavFlight/inav-configurator#1706] [Q&A]
Just wondering as stated in title, any possibility of Win 32 bit version?
Thanks

Cory

**Comment by b14ckyy** [comment]
there is 
![image](https://user-images.githubusercontent.com/33039058/220924847-2c911bf5-50e8-4478-a000-a77fed5bfce0.png)
